compiler/optimizer/InductionVariable.cpp

```diff
@@ -689,7 +689,7 @@ int32_t TR_LoopStrider::detectCanonicalizedPredictableLoops(TR_Structure *loopSt
                   }
 
                ListIterator<TR::Block> bi(&exitBlocks);
-               vcount_t visitCount = comp()->incVisitCount();
+               visitCount = comp()->incVisitCount();
```

Comment:
what a monster function. Isn't this the actual definition of the visitCount that's otherwise used in the function? I see the def is higher up and sets to zero but this spot looks like it dominates to me.